### ISSUE TITLE
Build error

### ISSUE THREAD

**Issue by behzaadh** [barche/jlqml#20]
Hello,

I am getting the following error during the compiling library: 
`libcxxwrap-julia\include\jlcxx\attr.hpp(139,27): error C2027: use of undefined type 'jlcxx::detail::process_attribute<bool,void>'`
Do you think it is because the master version of libcxxwrap-julia?

**Comment by behzaadh** [comment]
Apparently the library is not compatible with latest version of libcxxwrap-julia

**Comment by behzaadh** [comment]
By fixing the compatibility issue between libcxxwrap-julia and CxxWrap, I managed to compile.  By setting up Overrides.toml, everything works as it should work. However, I don't know why I am not able to run [QML.jl](https://github.com/JuliaGraphics/QML.jl/blob/main/src/QML.jl). I get the following error `ERROR: InitError: Unregistered method with key (:constructor, :QObject, :QML, 0x449b7e5b89ee04fd) requested, maybe you need to precompile the Julia module?`

**Comment by barche** [comment]
Not sure what the last error is about, possibly still an incompatibility between libcxxwrap and CxxWrap.jl versions. I have committed a fix for jlqml to build against the latest libcxxwrap, it should now be possible to use this with `CxxWrap#testjll` and by setting the QML.jl  CxxWrap compat to 0.15.

**Comment by behzaadh** [comment]
I don't think it is because of an incompatibility between libcxxwrap and CxxWrap.jl versions since I am able to run [QML examples](https://github.com/JuliaGraphics/QML.jl) but not be able to export packages [QML.jl](https://github.com/JuliaGraphics/QML.jl/blob/main/src/QML.jl). Actually, even without compiling and using the pre-compiled package, it gives the same error.

**Comment by behzaadh** [comment]
I realized that the function called `_get_function_pointer(mkey)` inside [CxxWrap.jl](https://github.com/JuliaInterop/CxxWrap.jl/blob/main/src/CxxWrap.jl) gives the aforementioned error:
`function _get_function_pointer(mkey)
  if !haskey(__global_method_map, mkey)
    error("Unregistered method with key $mkey requested, maybe you need to precompile the Julia module?")
  end
  return __global_method_map[mkey]
end`
